src/pages/new-claim/claim-evidence.tsx

```diff
@@ -0,0 +1,56 @@
+import Button from '../../components/button';
+import styles from './claim-evidence.module.scss';
+
+interface Props {
+  onNext: () => void;
+}
+
+export default function ClaimEvidence(props: Props) {
+  return (
+    <div className={styles.container}>
+      <p>Your claim must include the following:</p>
+      <ul className={styles.terms}>
+        <li>
+          a) Show a direct connection between the usage of your subscribed data feed and the claimed malfunction,
+          including on-chain evidence of affected transactions
```

Comment:
What if the problem is a non existing data-feed update? In that case there is no transaction on chain that can be used as proof.

Reply:
What is a non-existing data feed update? 🙂